src/playground-code-editor.ts

```diff
@@ -114,6 +63,29 @@ export class PlaygroundCodeEditor extends LitElement {
         /* Pretty much any color from the theme is good enough. */
         color: var(--playground-code-keyword-color, #770088);
       }
+
+      .diagnostic {
+        /* It would be nice to use "text-decoration: red wavy underline" here,
+           but unfortunately it renders nothing at all for single characters.
+           See https://bugs.chromium.org/p/chromium/issues/detail?id=668042. */
+        border-bottom: var(--playground-error-border, 2px red dashed);
```

Comment:
Q: this seems to layout fine, but I wondered if `outline` was the correct property to use.?